proto/p4/p4runtime.proto

```diff
@@ -168,6 +168,18 @@ message TableEntry {
   // default action of the table. If true, the "match" field must be empty and
   // the "action" field must be populated with a valid direct action.
   bool is_default_action = 8;
+  // The TTL for the entry, in nanoseconds. A value of 0 means that the table
+  // entry never "expires".
+  int64 idle_timeout_ns = 9;
```

Comment:
Is this field meaningful in a table read request?

Can a target implementation reject idle_timeout_ns values that are too small for hardware support?

Reply:
For a read request, the server should return the value provided by the client when writing the entry.

Regarding supported `idle_timeout_ns` values, I think we should go with "best effort" for the target, instead of failing. In other words, the target should approximate the value as best as possible (kind of like for meter specs). I can add this to the spec.